book/07-git-tools/sections/rewriting-history.asc

```diff
@@ -69,7 +69,7 @@ It may be easier to remember the `~3` because you're trying to edit the last thr
 $ git rebase -i HEAD~3
 ----
 
-Remember again that this is a rebasing command -- every commit included in the range `HEAD~3..HEAD` will be rewritten, whether you change the message or not.
+Remember again that this is a rebasing command -- every commit included in the range `HEAD~3..HEAD` will be rewritten if you change any message.
```

Comment:
Technically this isn't entirely true - if you change the message on `HEAD~1`, this will only rewrite `HEAD~1` and `HEAD`. Maybe say this instead?

```suggestion
Remember again that this is a rebasing command -- every commit in the range `HEAD~3..HEAD` with a changed message _and all of its descendants_ will be rewritten.
```

Reply:
Thank you! I didn't notice that. I'll use your words.